src/amo/components/HomeHeroBanner/styles.scss

```diff
@@ -42,33 +42,35 @@ $colors-third-item: ('cyan', #15aabf, #1098ad) ('teal', #12b886, #0ca678);
 }
 
 .HeroSection {
```

Comment:
```suggestion
.HeroSection-styleName--default {
```

Reply:
Maybe `default` is not a great name anymore. We could change it in the `Hero` component to: `random-color` maybe?

Reply:
yeh I agree. `random-color` sounds fine to me...